.github/workflows/deploy-docs.yml

```diff
@@ -0,0 +1,37 @@
+name: Deploy docs
+on:
+  push:
+    branches: main
+    paths:
+      - docs/**
+      - src/**
+      - .github/workflows/deploy-docs.yml
+  workflow_dispatch:
+concurrency:
+  group: ${{ github.workflow }}
+  cancel-in-progress: true
+jobs:
+  deploy-docs:
+    permissions:
+      contents: read
+      id-token: write
+      pages: write
+      deployments: write
+    environment:
+      name: github-pages
+      url: ${{ steps.deploy-pages.outputs.page_url }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+      - run: cd docs && npm ci
+      - id: configure-pages
+        uses: actions/configure-pages@v3
+      - run: cd docs && npm pack
+        env:
+          BASE_URL: ${{ steps.configure-pages.base_url }}/

```

Comment:
What is `steps.configure-pages.base_url`?
Maybe `steps.deploy-pages.outputs.page_url`?